iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_receiver.hpp

```diff
@@ -34,6 +34,23 @@ enum class ChunkReceiveResult
     NO_CHUNK_AVAILABLE
 };
 
+/// @brief Converts the ChunkReceiveResult to a string literal
+/// @param[in] value to convert to a string literal
+/// @return pointer to a string literal
```

Comment:
I think a note is not really necessary since to compiler will generate an error when the enum is extended and not added to the `asStringLiteral` function, similar to other cases where an `enum class` without a default value is used. This is also the advantage compared to just using an array of strings.

If you insist, I can add the note tough.